test/integration/utils.go

```diff
@@ -96,3 +100,68 @@ func GetEtcdClients(config storagebackend.TransportConfig) (*clientv3.Client, cl
 
 	return c, clientv3.NewKV(c), nil
 }
+
+func StartTestAPIServer(t *testing.T) (server *kubeapiservertesting.TestServer, apiserverConfig *os.File, token string, err error) {
```

Comment:
why this change? 
we have many common methods of starting an apiserver,  the goal is to reduce them, if this is only used by one test is better to keep it that way instead of exposing it and eventually another test uses it, making harder to remove it later

Reply:
Initially, I want to deduplicate the code which I need to reuse. But your comment makes sense and I will just copy them in a private helper function to make it less difficult for future removals.